templates/repo/issue/view_content/pull.tmpl

```diff
@@ -131,38 +131,37 @@
 		{{$showGeneralMergeForm := false}}
 		<div class="ui attached merge-section segment {{if not $.LatestCommitStatus}}no-header{{end}}">
 			{{if .Issue.PullRequest.HasMerged}}
-				<div class="item text">
-					{{if .Issue.PullRequest.MergedCommitID}}
-						{{$link := printf "%s/commit/%s" $.Repository.Link (.Issue.PullRequest.MergedCommitID|PathEscape)}}
-						{{if eq $.Issue.PullRequest.Status 3}}
-							{{$.locale.Tr "repo.pulls.manually_merged_as" ($link|Escape) (ShortSha .Issue.PullRequest.MergedCommitID) | Safe}}
-						{{else}}
-							{{$.locale.Tr "repo.pulls.merged_as" ($link|Escape) (ShortSha .Issue.PullRequest.MergedCommitID) | Safe}}
-						{{end}}
-					{{else}}
-						{{$.locale.Tr "repo.pulls.has_merged"}}
+				<div class="item item-section text gt-f1">
+					<div class="item-section-left">
+						<h3 class="gt-mb-3">{{$.locale.Tr "repo.pulls.merged_success"}}</h3>
+						<div class="merge-section-info">
+							{{$.locale.Tr "repo.pulls.merged_info_text" (printf "<code>%s</code>" .HeadTarget) | Str2html}}
```

Comment:
`HeadTarget` needs to be escaped.

Reply:
And one more thing (I haven't checked), is `HeadTarget` the same as `MergedCommitID` in all cases?

Reply:
idk either, I am just altering existing rendering.